docker/autocut-with-trained-model/config.yaml

```diff
@@ -1,14 +1,24 @@
 steps:
 - name: 'gcr.io/cloud-builders/docker'
-  args: ['build',
-         '--build-arg',
-         'base_image=${_BASE_IMAGE}',
-         '-t',
-         '${_IMAGE}',
-         '.']
+  entrypoint: 'bash'
+  args: [
+    '-c',
+    'if [ ! -z "${_DOCKER_HUB_USERNAME}" ]; then docker login --username="${_DOCKER_HUB_USERNAME}" --password="${_DOCKER_HUB_PASSWORD}"; fi'
```

Comment:
is the `_` leading prefix to mark them as sensitive or private or ...?

Reply:
This is to distinguish from builtin variables. See [docs](https://cloud.google.com/cloud-build/docs/configuring-builds/substitute-variable-values):

> Substitutions must begin with an underscore (_) and use only uppercase-letters and numbers (respecting the regular expression _[A-Z0-9_]+). This prevents conflicts with built-in substitutions.